### ISSUE TITLE
Add topo scale feature for large mega macro topos to make features smaller

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#1701]
From Julian S:

> The belay/anchor points in the current topos are what I was talking about. Obviously you need to indicate the pitch ending, but the current 'dot' is huge. On a distant photo topo Perry's the dots and lines conceal quite a lot of 'wall space'. As such, you lose perspective and, moreover, any appeal the wall might have. For instance, there are now quite a few pitches to the right of Red Edge (Perry's Lookdown) at pitches 3, 4 and 5. (http://www.thecrag.com/climbing/australia/blue-mountains/perrys-lookdown/route/13161403). If I put them in it pretty much covers the entire wall; literally blanks it out. 
> 
> The lines need to be about the width of the black edging of the current lines, or thinner, on both the distant and the zoomed in version. 
- [ ] need to add topo field for storing a scale factor, most topos will be 1 but larger topos could be .8 or .5. This scale factor applies to all drawn features, effectively exactly the same as if the topo was made 2 times bigger and then responsively scaled down
- [ ] add atomTopo template data for scale
- [ ] add extra field to api endpoint for saving it
- [ ] wire in to topo JS (already working)

**Comment by brendanheywood** [comment]
test case: https://www.thecrag.com/climbing/australia/bungonia-gorge/area/11771389/topo

**Comment by killakalle** [comment]
@scd @rouletout 
After enabling high-quality photos in one of the recent releases, wouldn't it be a logical next step to think about the scaling of the topo features and icons? Just thinking out loud.

**Comment by scd** [comment]
Yup. What you don't know is that we have somebody new taking over maintenance of topo code and he is building a new version of the topo. 

The photo res is something I did with the old code because it was fairly easy, but the other scaling issues is way more difficult so I did not take it on considering we have a new version coming out sometime. 

We are now really stuck for topo improvements until the new version comes out.

**Comment by killakalle** [comment]
An interesting observation:

Apparently different scaling of topo features is used depending on whether the same topo is displayed on an area vs on a route detail page.

See these examples:
https://www.thecrag.com/es/escalar/spain/alto-mijares/casucho/area/587201946
https://www.thecrag.com/es/escalar/spain/alto-mijares/casucho/route/870992673

On the area the topo features are too big and it's messy:
![image](https://github.com/theCrag/website/assets/30682796/82b2b4a3-04ca-4314-a20f-c19a1741e9e6)

On the route detail the topo features have a nice scale
![image](https://github.com/theCrag/website/assets/30682796/33dbb1ca-b228-4566-8ac7-da2ed43fbad4)

When editing the topo (from the route detail view), it shows again the bigger (messy) features
![image](https://github.com/theCrag/website/assets/30682796/4817036b-9c50-4cf2-83b7-c8e97550c135)


@scd 
Might it be worth exploring to use the same method for topos from the route detail view on the area view (and also when drawing topos)

**Comment by killakalle** [comment]
I'm now using the following workaround:

I try to make topo photos as "flat" as possible by cutting away landscape above and below the routes. This results in a wider display of the topo images. As a side-effect the topo lines are also drawn finer on wider topo images.

![image](https://github.com/theCrag/website/assets/30682796/694ed8d3-1f58-4eb2-b335-8dfcdf3e8eef)

Also related to https://github.com/theCrag/website/issues/3415

**Comment by killakalle** [comment]
There is actually an interesting effect when you look at the same topo on the crag level vs on the route level.

On the crag level, features are drawn thicker. This often results in the number boxes overlaying each other. If you look at the same topo image on a route level, features are drawn thinner and you can most times read the individual number fields.

![image](https://github.com/theCrag/website/assets/30682796/85539d01-a179-4347-8390-f68991880378)

- Crag: https://www.thecrag.com/es/escalar/spain/bellus/area/2024008017
-  Route: https://www.thecrag.com/es/escalar/spain/bellus/route/2730397734

I'm not a developer, but could it be already helpful to just apply the same settings as used on the route detail page also on the crag overview page?

![image](https://github.com/theCrag/website/assets/30682796/a9ddd179-4d09-4149-ab13-afbf57cd2817)
